File: patch_catalog.py
import re

with open("/opt/elektrix/apps/api/modules/storefront/catalog.py", "r") as f:
    c = f.read()

c = c.replace(
    '"""(c.slug = :cat OR c.id::text = :cat OR c.parent_id IN (\n                    SELECT id FROM categories WHERE slug = :cat))"""',
    '"""(c.slug = :cat OR c.id::text = :cat OR c.parent_id IN (SELECT id FROM categories WHERE slug = :cat) OR (p.tags IS NOT NULL AND p.tags::text ILIKE :cat_tag))"""'
)

with open("/opt/elektrix/apps/api/modules/storefront/catalog.py", "w") as f:
    f.write(c)

File: patch_catalog.js
const fs = require('fs');
const file = '/opt/elektrix/apps/api/modules/storefront/catalog.py';
let content = fs.readFileSync(file, 'utf8');

content = content.replace(
  'p.slug, p.description, p.brand, p.sku, p.price, p.mrp,',
  'p.slug, p.description, p.brand, p.return_policy, p.warranty_info, p.sku, p.price, p.mrp,'
);

content = content.replace(
  'brand=d.get("brand"),',
  'brand=d.get("brand"),\n            return_policy=d.get("return_policy"),\n            warranty_info=d.get("warranty_info"),'
);

fs.writeFileSync(file, content);
